### PR TITLE
add missing parameters

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/9771

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsChannelsPolicy.md [hunk 1]
```diff
@@ -158,6 +159,51 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowChannelSharingToExternalUser
+Owners of a shared channel can invite external users to join the channel, if Azure AD external sharing policies are configured. If the channel has been shared with an external member or team, they will continue to have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsChannelsPolicy.md:163]:
```suggestion
Owners of a shared channel can invite external users to join the channel if Azure AD external sharing policies are configured. If the channel has been shared with an external member or team, they will continue to have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```

File: skype/skype-ps/skype/New-CsTeamsChannelsPolicy.md [hunk 2]
```diff
@@ -158,6 +159,51 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowUserToParticipateInExternalSharedChannel
+Users and teams can be invited to external shared channels, if Azure AD external sharing policies are configured. If a team in your organization is part of an external shared channel, new team members will have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsChannelsPolicy.md:193]:
```suggestion
Users and teams can be invited to external shared channels if Azure AD external sharing policies are configured. If a team in your organization is part of an external shared channel, new team members will have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```

File: skype/skype-ps/skype/Set-CsTeamsChannelsPolicy.md [hunk 3]
```diff
@@ -183,6 +185,48 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowChannelSharingToExternalUser
+Owners of a shared channel can invite external users to join the channel, if Azure AD external sharing policies are configured. If the channel has been shared with an external member or team, they will continue to have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsChannelsPolicy.md:189]:
```suggestion
Owners of a shared channel can invite external users to join the channel if Azure AD external sharing policies are configured. If the channel has been shared with an external member or team, they will continue to have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```

File: skype/skype-ps/skype/Set-CsTeamsChannelsPolicy.md [hunk 4]
```diff
@@ -183,6 +185,48 @@ Accept pipeline input: False
...
+Aliases:
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowUserToParticipateInExternalSharedChannel
+Users and teams can be invited to external shared channels, if Azure AD external sharing policies are configured. If a team in your organization is part of an external shared channel, new team members will have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsChannelsPolicy.md:217]:
```suggestion
Users and teams can be invited to external shared channels if Azure AD external sharing policies are configured. If a team in your organization is part of an external shared channel, new team members will have access to the channel even if this parameter is set to FALSE. For more information, see [Manage channel policies in Microsoft Teams](/microsoftteams/teams-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**